src/Centreon/Application/Controller/PlatformTopologyController.php

```diff
@@ -145,6 +151,14 @@ public function getPlatformJsonGraph(): View
     {
         $this->denyAccessUnlessGrantedForApiConfiguration();
 
+        // Check Topology access to Configuration > Pollers page
+        if (
+            !$this->getUser()->hasTopologyRole(Contact::ROLE_CONFIGURATION_MONITORING_SERVER_READ)
+            && !$this->getUser()->hasTopologyRole(Contact::ROLE_CONFIGURATION_MONITORING_SERVER_WRITE)
```

Comment:
Are both roles mandatory?
```suggestion
            || !$this->getUser()->hasTopologyRole(Contact::ROLE_CONFIGURATION_MONITORING_SERVER_WRITE)
```